skimage/filters/rank/bilateral.py

```diff
@@ -93,7 +93,7 @@ def mean_bilateral(
 
     See also
     --------
-    denoise_bilateral
+    :func:`skimage.restoration.denoise_bilateral`
```

Comment:
```suggestion
    skimage.restoration.denoise_bilateral
```
The `See also` section is special and the markup shouldn't be necessary here as long as the path is specific enough. Might even get away with `~.denoise_bilateral` but haven't tested that and I think the full path is the better option.